### PR TITLE
Batch Email Notifications

### PR DESCRIPTION
**Pull request by laritakr**
Refs https://github.com/scientist-softserv/palni_palci_knapsack/issues/121

`last_emailed_at` was previously added to the User table. However User table is not tenant-specific, and this date needed to be unique to each tenant. This moves the `last_emailed_at` date to its own table and modifies the use to match the new location.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -407,6 +407,12 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "hyrax_flexible_schemas", force: :cascade do |t|
```
Comment (laritakr) [inline db/schema.rb:410]: I'm not sure why this came in... there must have been a prior migration that hadn't been run and saved?

**Review by sjproctor — APPROVED** [review]
Looks good to me!

File: db/schema.rb (hunk 1 above)
Comment (orangewolf) [reply db/schema.rb:410]: It needs to be manually removed or it will break future migrations on new apps. Just deleting these lines will do the trick.

**Review by orangewolf — CHANGES_REQUESTED** [review]
This is good to go other than the db/schema.rb change.

**Review by kirkkwang — APPROVED** [review]

**Pull request merged**